### PR TITLE
Properly format Pokemon with event abilities in /data

### PR DESCRIPTION
**Pull request by asgdf**
To be specific, to fix this: 
grafik

### CODE REVIEW

File: js/battledata.js [hunk 1]
```diff
@@ -486,22 +486,17 @@ var Tools = {
...
-				if (i == 'H') {
-					ability = '</span><span class="col abilitycol"><em>' + (template.unreleasedHidden ? '<s>' + ability + '</s>' : ability) + '</em>';
-				}
-				buf += ability;
+			if (template.abilities['S']) {
+				buf += '<span class="col twoabilitycol' + (template.unreleasedHidden ? ' unreleasedhacol' : '') + '"><em>' + template.abilities['H'] + '<br />' + template.abilities['S'] + '</em></span>';
+			} else if (template.abilities['H']) {
+				buf += '<span class="col abilitycol' + (template.unreleasedHidden ? ' unreleasedhacol' : '') + '"><em>' + template.abilities['H'] + '</em></span>';
+			} else {
+				buf += '<span class="col abilitycol" />';
```
Comment (Zarel) [inline js/battledata.js]: This is HTML, not XML; `<span />` is invalid code; you need to write `<span></span>`

Comment (Zarel) [inline js/battledata.js]: `<span />` is actually treated as `<span>` (an unclosed tag) by HTML, unfortunately.

**Comment by asgdf** [comment]
TIL, didn't know that. I assumed it worked for all empty elements. Anyway, fixed.

**Comment by Zarel** [comment]
Yeah, I guess it's better this way for explicitness...

**Pull request merged**